javascript/example_code/iam/iam_createaccountalias.js

```diff
@@ -23,7 +23,7 @@
 //snippet-sourceauthor:[AWS-JSDG]
 
 // ABOUT THIS NODE.JS SAMPLE: This sample is part of the SDK for JavaScript Developer Guide topic at
-// https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/iam-examples-account-alises.html
+// https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/iam-examples-account-aliases.html

```

Comment:
This change (and the other two like it) break the link. Are you planning to also fix the URL in the dev guide (and provide a redirect)?

Reply:
Already done.